lr-cli/cortx_setup/commands/cluster/config/component.py

```diff
@@ -37,10 +37,9 @@ class ClusterConfigComponent(Command):
             'optional': True,
             'default': None,
             'dest': 'component_group',
-            'choices': ['foundation', 'iopath', 'controlpath', 'ha'],
+            'choices': ['prerequisites', 'foundation', 'iopath', 'controlpath', 'ha'],
```

Comment:
Lets change the choices as:
prerequisites, io_path, control_path, ha_path

`foundation` should be merged into `prerequisites`

Reply:
@nitin-seagate 
Prerequisites is 3rd party components and we don't have any mini provisioner for them , so we are running all states for 3rd party in ` cortx_setup cluster config` command, but foundation is part of cortx component it has mini provisioner available so we need to run it's post_install, prepare , config and init separately just like other cortx components.

so I think it is better to keep that component as separate like foundation. which is part of cortx components.

Reply:
I'm proposing choices presented to the user, internally we can install 3rdparty and foundation as part of `prerequisites` component group...
Next please change the user choices as:
`prerequisites, io_path, control_path, ha_path`